src/components/Navbar.jsx

```diff
@@ -0,0 +1,23 @@
+import logo from "../assets/images/logo.svg";
+import cart from "../assets/images/icon-cart.svg";
+import avatr from "../assets/images/image-avatar.png";
```

Comment:
Great job, Reda! 🎉
However, we need to fix a typo here.
It should be `avatar` instead of `avatr`